index.html

```diff
@@ -92,7 +92,7 @@ <h2>Schedule Commander</h2>
                 <!-- There is no official MDL selector, so need to find a working and nice element for this. -->
                 <div class="mdl-cell mdl-cell--12-col">
                     <label for="label-color">Label Color</label>
-                    <select id="label-color" class="u-full-width">
+                    <select id="label-color" class="u-full-width mdl-textfield__input" style="border-radius: 8px;">
```

Comment:
For style changes, please include before/after screenshots and a description of the change.

This is a good change and I would welcome just this as PR of its own: Updating the selector to use the MDL style that is now available is a substantial fix over the current default selector. However, please leave out the border radius change, as none of the other elements have borders that round -- unless you have an idea for applying rounded borders as a consistent design overhaul to the rest of the application.

Reply:
Ok, I will make that change.